### ISSUE TITLE
A11y_Visual Studio Code Web Extensions _GitHub Pull request_ Create Pull request_ Screen Reader: Narrator is not announcing the state of Expanded/collapsed for "Create with Option" arrow button

### ISSUE THREAD

**Issue by tripathiakhil429** [microsoft/vscode-pull-request-github#5483]
**Actual:**

 Narrator is not announcing the state of Expanded/collapsed for "Create with Option" arrow button
**Note: Similar issue is observed with NVDA also.**

**Expected**: 

 Narrator should announce the state of Expanded/collapsed for "Create with Option" arrow button

**GitHubTags:** #A11yMAS;#A11yTCS;#BM_Visual Studio Code Web Extensions_Nov2023;#Visual Studio Code Web Extensions;#ChromiumEdge;#DesktopWeb;#A11ySev2;#Win11;#WCAG1.3.1;#Screen reader;#Narrator; #NVDA;#AIlimited

**Environment Details**
Application Name: Visual Studio Code Web Extensions
URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github
Operating System: Windows 11 Enterprise Insider Preview(23H2)
OS Build: 25997.1000
Edge version: 120.0.2210.7 (Official build) stable app, dev channel (64-bit)
Screen Reader: Narrator/NVDA
NVDA Version: 2022.4(2022.4.0.27401)

**Repro Steps:**

1.Turn on Narrator/NVDA
2.Hit the URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github and login with git hub account.
3.Tab to "More Options" button present in the left navigation panel.
4.Tab to GitHub Pull request and press enter.
5.Tab to Create Pull request and press enter
6.Tab to "Create with Option" arrow button.
7.Observe Narrator is not announcing the state of Expanded/collapsed for "Create with Option" arrow button

**User Impact:**

Screen reader only users will not get the information about the Create with Option arrow button

**MAS Reference:**

[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Attachment:**

[5483 A11y_Visual Studio Code Web Extensions _GitHub Pull request_ Create Pull request_ Screen Reader Narrator is not announcing the state of Expandedcollapsed for Create with Option arrow button.webm](https://github.com/microsoft/vscode-pull-request-github/assets/85679252/d94f0f39-e890-4576-aec9-a04032867653)

[5483 A11y_Visual Studio Code Web Extensions _GitHub Pull request_ Create Pull request_ Screen Reader NVDA is not announcing the state of Expandedcollapsed for Create with Option arrow button.webm](https://github.com/microsoft/vscode-pull-request-github/assets/85679252/3f47db06-3c5f-4a2d-b44c-9d7a5bedd09b)

**Comment by alexr00** [comment]
I do see that `menu` is read when the menu is opened using the "create with option" button

![image](https://github.com/microsoft/vscode-pull-request-github/assets/38270282/d55bdb02-fe34-486e-b8b9-15c3168f56fe)

Is there something else that should be read?

**Comment by rzhao271** [comment]
Does the expand and collapse button contain an aria-expanded attribute?
Ref https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#buttons

**Comment by rzhao271** [comment]
Confirmed that that accessibility attribute isn't there. Reopening.

**Comment by alexr00** [comment]
It's actually a menu, so I think that only the expand needs to be there: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#menu
